### PR TITLE
Publish Stash 0.7.0-rc.3 chart

### PR DESCRIPTION
**Pull request by 1gtm**
https://github.com/appscode/stash/releases/tag/0.7.0-rc.2

### CODE REVIEW

File: stable/stash/README.md
Comment (timstoop) [inline stable/stash/README.md]: Why add this? It's not used in the code.

File: stable/stash/templates/deployment.yaml
Comment (timstoop) [inline stable/stash/templates/deployment.yaml:44]: Enabling this would require a bit more heads-up! Make sure there's a note in the README about it and a description of the enableAnalytics variable. This would also warrant a bump of the minor version of this Chart, as it adds a new feature. I'd prefer to see this handled in a separate PR.

File: stable/stash/values.yaml
Comment (timstoop) [inline stable/stash/values.yaml]: Again, this variable is not used in the code, so I see no reason to add it.

File: stable/stash/values.yaml
Comment (timstoop) [inline stable/stash/values.yaml:55]: I'm not sure how people will react to Chart defaults being "share my usage pattern". I'd say, default to false and let the user decide. At the very least, make a note about it in the README.

**Review by timstoop — CHANGES_REQUESTED** [review]
It looks to me like you were preparing another PR as well and it got added to this one?

File: stable/stash/templates/deployment.yaml
Comment (1gtm) [reply stable/stash/templates/deployment.yaml:44]: Done.

File: stable/stash/values.yaml
Comment (1gtm) [reply stable/stash/values.yaml:55]: We want this to be default true. If users don't want that they can set false.

Comment (unguiculus) [reply stable/stash/values.yaml:55]: I agree with @timstoop. This should be disabled by default.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/stash/values.yaml
Comment (tamalsaha) [reply stable/stash/values.yaml:55]: I disagree. Application developers should be able to understand how their apps are used. Analytic is a decision between app developers and users. I think chart repo has no business in that.

**Comment by prydonius** [comment]
It sounds like we need to have a deeper discussion around the use of tracking in charts. I suggest we have this discussion at the next Office Hours (next Tues 10th April @ 9am PT) cc @kubernetes/charts-maintainers.

/hold

**Comment by tamalsaha** [comment]
We have decided to host our charts at https://github.com/appscode/charts

**Pull request closed without merging**
